nilearn/datasets/neurovault.py

```diff
@@ -1507,11 +1513,71 @@ def _download_image_nii_file(image_info, collection, download_params):
         collection['relative_path'], image_file_name)
     image_absolute_path = os.path.join(
         collection['absolute_path'], image_file_name)
-    _simple_download(
-        image_url, image_absolute_path,
-        download_params['temp_dir'], verbose=download_params['verbose'])
-    image_info['absolute_path'] = image_absolute_path
-    image_info['relative_path'] = image_relative_path
+
+    # Testing whether we need to resample or not
+    # The following behavior is intended :
+    # If resampling is needed,
+    # - check whether the resampled images are already on disk
+    #       - if yes, return them
+    #       - if not, check whether the non-resampled image is on disk
+    #           - if yes : use this image and resample it , keep the non resampled version
+    #           - if not : download the image as a tempfile, resample it, remove the tempfile
+
+
+    if download_params['resample']:
+
+
+
+        resampled_image_file_name = 'image_{0}_resampled.nii.gz'.format(image_id)
```

Comment:
Why not `'{0}_resampled.nii.gz'.format(image_id)`

Reply:
I imagine to be similar to the original image file names, which are stored as `'image_{}.nii.gz'.format(image_id)`, where `image_id` is an `int`: image_123.nii.gz, image_123_resampled.nii.gz